.github/workflows/jira.yml

```
@@ -0,0 +1,34 @@
+name: Jira
+
+on:
+  pull_request:
+    branches: [main]
+    paths-ignore: ["*.md"]
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check Jira Story
+        shell: pwsh
+        run: |
+          Install-Module JiraPS -Scope CurrentUser -Force
+          if ("$env:GITHUB_HEAD_REF" -like "*renovate/*" -eq $True)
+          {
+            Write-Host "Skipping, renovate branch detected"
+            return
+          }
+
+          Set-JiraConfigServer -Server "${{ secrets.JIRA_URL }}"
+          if("$env:GITHUB_HEAD_REF" -match "IDP-[0-9]+" -eq $False)
```

Should we already put it tis way?
```suggestion
          if("$env:GITHUB_HEAD_REF" -match "[A-Za-z]-[0-9]+" -eq $False)
```

Actually I think we could have those workflows in another repo and reuse them, but maybe or anoter story

---

Your regex is incorrect, it only matches one letter x) Do you want me to fix it?

---

woops, should be fixed 😅